release/prologue.sh

```diff
@@ -27,11 +27,15 @@ LOG_INDENT=1 fetch_config_repo
 
 rm -rf "${WORK_DIR}"
 mkdir -p "${WORK_DIR}"/release/{config,images}
+
 mkdir -p "${PKGS_DIR}"
 
 mkdir -p "${MIXER_DIR}"
 rm -rf "${MIXER_DIR}/local-yum"
 mkdir -p "${MIXER_DIR}/local-yum"
+if function_exists sign_update; then
```

Comment:
I think is better to check if SWUPD_CERT is not Null/Undefined than to rely on the function implementation.